test/integration/targets/assert/runme.sh

```diff
@@ -22,8 +22,8 @@ run_test() {
       > >(set +x; tee "${OUTFILE}.${testname}.stdout"); } \
       2> >(set +x; tee "${OUTFILE}.${testname}.stderr" >&2) 0</dev/null
    # clean up Python 2 vs 3 differences
-   sed -i 's/[[:blank:]]*$//' "${OUTFILE}.${testname}.stdout"
-   sed -i 's/[[:blank:]]*$//' "${OUTFILE}.${testname}.stderr"
+   sed -i 's/ *$//' "${OUTFILE}.${testname}.stdout"
+   sed -i 's/ *$//' "${OUTFILE}.${testname}.stdout"
```

Comment:
Now you're processing the stdout file twice (instead of once stdout and once stderr).

Reply:
Yes, and apparently -i requires a suffix on OSX.

Reply:
It doesn't work that simply, though. On OSX, you need to do `sed -i '' 's/...` (note the space between `-i` and `''`), but that won't work under Linux.